<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Dr. Yanjun Qi</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="Yanjun Qi" >

    <link href="style.css" rel="stylesheet" type="text/css"> 
    <link href="./bootstrap/css/bootstrap.css" rel="stylesheet">
    <script src="./bootstrap/js/bootstrap.min.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js"></script>

</head>


<body>

<p><a name="topPage"></a></p>

    <!-- Begin topbar  -->

<div class="container"> 

<nav class="navbar navbar-fixed-top navbar-expand-sm navbar-dark bg-info">

  <a class="navbar-brand" href="index.htm">Dr. Yanjun Qi</a>

 <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button> 

  <div class="collapse navbar-collapse" id="navbarSupportedContent">
    <ul class="navbar-nav">
      <li class="nav-item ">
        <a class="nav-link" href="papers.htm" >Publications</a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="teaching.htm" >Teaching </a>
      </li>

      <li class="nav-item">
        <a class="nav-link" href="gQdata.htm" >Research </a>
      </li>

      <li class="nav-item dropdown">
        <a class="nav-link dropdown-toggle" href="gnews.htm" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
          Group
        </a>
        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
          <a class="dropdown-item"  href="https://qdata.github.io/qdata-page/"
        target="_blank">Group Research Blog</a>
          <a class="dropdown-item"  href="https://github.com/qdata" target="_blank">Group GitHub</a>

          <a class="dropdown-item"  href="https://github.com/Qdata4Capstone" target="_blank">Group Capstone GitHub</a>

          <a class="dropdown-item"  href="https://www.youtube.com/channel/UCHMYETgeGbNHVHLidZSV8BQ" target="_blank">Group Research Videos</a>
          <div class="dropdown-divider"></div>
          <a class="dropdown-item"  href="https://qdata.github.io/qdata-page/member-page/" target="_blank">Group Members</a>
          <a class="dropdown-item"  href="https://qdata.github.io/qdata-page/news/" target="_blank">Group News</a>


          <div class="dropdown-divider"></div>
          <a class="dropdown-item"  href="https://qdata.github.io/deep2Read/aReadingsIndexByDate/" target="_blank">Journal Club: Algorithm</a>
          <a class="dropdown-item"  href="https://qdata.github.io/deep-learning-undergrad-reading-group/" target="_blank">Journal Club: Undergraduate</a>

          <div class="dropdown-divider"></div>
          <a class="dropdown-item"  href="http://qiyanjun.github.io/books2read//tag/" target="_blank">My Notes on Readings</a>
          <div class="dropdown-divider"></div>
          <i><b>OLD legacy research sites</b></i>
          <a class="dropdown-item"  href="http://deeplearning4discrete.net" target="_blank">deeplearning4discrete.net</a>
          <a class="dropdown-item"  href="http://deeplearning4biomed.org" target="_blank">deeplearning4biomed.org</a>
          <a class="dropdown-item"  href="http://jointggm.org" target="_blank">jointnets.org</a>
           <a class="dropdown-item"  href="http://securemachinelearning.org" target="_blank">TrustworthyMachineLearning.org</a>
        </div>
      </li>

      <li class="nav-item">
        <a class="nav-link"
        href="http://scholar.google.com/citations?user=eXKdSu0AAAAJ&hl=en"
        target="_blank" >GoogleScholar</a>
      </li>

      <li class="nav-item">
          <a class="nav-link"  href="bio.htm">ShortBio</a>
      </li>

      <li class="nav-item">
          <a class="nav-link"  href="contact.htm">Contact</a>
      </li>


            
    </ul>
  </div>
</nav>       
</div>  
<!-- End topbar -->



    <!-- Begin Container -->
 <div class="container">

      <!-- #BeginEditable "content" -->

<BR>

	<span class="style2">

<a name="top"></a>

<!--			<table cellpadding="3" cellspacing="5" style="width: 100%">
			  <tr>
			    <td style="width: 25%">
		  <h3 class="style7">
		  <img alt="" src="index_files/qdataWordC.png" height="85" border="0"></h3>
			    </td>
			    <td style="width: 55%">
			      <h2 ><font color="red">“Machine Learning for Big Data Analytics In Biomedicine" </font></h2>
			    </td>
				</tr>
	      </table>
-->

     <div align="center"><span style="" lang="EN-US">
                       <hr align="left" size="1">      </span></div>

 <h4><font color="brown"><font size=+1>Basic Information:</font></font></h4>

<a name="info"></a>

	      <table cellpadding="3" cellspacing="5" >
			<tr> <td>UVA "Machine Learning QData" group focuses on developing novel
			  machine-learning techniques on important
			  challenges with medical impacts, especially those
			  dealing with complex data sets. We strive toward building and sharing benchmarked datasets and open-source research toolkits.
        <br>

  <br>
  <ul>
    <li> [ <a href="https://qdata.github.io/qdata-page/" target="_blank">Group Website</a>]
  </li>
  <li>
 [ <a href="https://github.com/qdata" target="_blank">Group GitHub</a>]
  </li>
   <li>
 [ <a href="https://qdata.github.io/deep2Read/aReadingsIndexByDate/" target="_blank">Group Deep-Learning Journal Club</a>]
  </li>


 </ul>
      </td> </tr>

	      </table>

<!-----
	  Taking an
			  inter-disciplinary approach, we have devoted
			  our research efforts on both  the advancement
			  of machine learning technologies and the improvements
			  of biomedical research.

  	<h4>Machine Learning and Data Mining </h4></span>
			  Rapidly-growing “Big Data” resources have posted many interesting new challenges for machine learning, with the data being large, loosely labeled, highly diverse, complex and often relationally structured. We have proposed a diverse range of machine learning approaches, including <font color="blue">feature learning, deep learning, sparse modeling, latent factor analysis,  semi-supervised learning, multi-task learning, secure machine learning, </font>and others, to handle different types of data complexities that are urgent to be addressed in this domain.

			 	<h4>Applications in Biomedicine and More </h4></span>
					  My group takes an inter-disciplinary approach, combining ideas from statistical machine learning, computational biology, computer architecture, secure computation and language technologies. Over years we have had valuable opportunities to work on many practically important applications, covering multiple research fields, such as proteomics, cancer genomics, medical informatics, bio-text mining, structural biology, immunology, more traditional text mining topics, like information extraction, ad hoc information retrieval and text/image/video labeling and recently on cloud system analysis and sensor data mining.
						</td>
				</tr>
-->


<div align="center"><span style="" lang="EN-US">
                       <hr align="left" size="1">      </span></div>

<a name="grantNow"></a>

 <h4><font color="brown"><font size=+1>Table of grant awards we are working on :</font></font></h4>

<a name="grant"></a>
<table id="datatab2" summary="Table of external grant awards we are working on: " border="1">
  <colgroup>
       <col span="1" style="width: 15%;">
       <col span="1" style="width: 10%;">
       <col span="1" style="width: 50%;">
       <col span="1" style="width: 15%;">
       <col span="1" style="width: 20%;">
  </colgroup>
<tbody><tr>
  <th>Name</th>
  <th>Role</th>
  <th>Summary</th>
  <th>Project Website</th>
  <th><a href="#grant" style="color:white;">Year</a></th>
</tr>

<tr>
  <td>NSF CNS-124538: </td>
  <td>PI</td>
  <td>SaTC: CORE: Small: Generalizing Adversarial Examples in Natural Language </td>
  <td> <a href='https://www.nsf.gov/awardsearch/showAward?AWD_ID=2124538&HistoricalAwards=false'>Project Website</a> </td>
  <td>2021-2024</td>
</tr>




<tr>
  <td>NSF CCF-1900676: </td>
  <td>co-PI</td>
  <td>SHF: Medium: Rearchitecting Neural Networks for Verification </td>
  <td> <a href='https://www.nsf.gov/awardsearch/showAward?AWD_ID=1900676'>Project Website</a> </td>
  <td>2019-2023</td>
</tr>

</tbody>
</table>



<a name="grant"></a>
<table id="datatab2" summary="Selected UVa internal grant awards we are working and worked on: " border="1">
  <colgroup>
       <col span="1" style="width: 50%;">
       <col span="1" style="width: 15%;">
  </colgroup>
<tbody><tr>
  <th>Selected UVa internal grant awards we are working and worked on:</th>
  <th><a href="#grant" style="color:white;">Year</a></th>
</tr>


<tr>
  <td>UVA Engineering-In-Medicine (EIM)  Research Innovation Awards</td>
  <td>2020-2021</td>
</tr>

<tr>
  <td>UVA Human and Machine Intelligence Seed Award</td>
  <td>2020</td>
</tr>



<tr>
  <td>UVA SEAS  Research Innovation Award</td>
  <td>2017-2018</td>
</tr>


<tr>
  <td>UVA Brain Institute Seed Award for Transformative, Collaborative Neuroscience Pilot Grant  </td>
  <td>2017-2018</td>
</tr>



<tr>
  <td>UVA SEAS Cybersec Initiative Seed Award </td>
  <td>2016-2017</td>
</tr>

</tbody>
</table>





     <div align="center"><span style="" lang="EN-US">
                       <hr align="left" size="1">      </span></div>

 <h4><font color="brown"><font size=+1>Table of external grant awards we have completed :</font></font></h4>

<a name="grantThen"></a>
<table id="datatab2" summary="Table of external grant awards we have completed: " border="1">
  <colgroup>
       <col span="1" style="width: 15%;">
       <col span="1" style="width: 10%;">
       <col span="1" style="width: 50%;">
       <col span="1" style="width: 15%;">
       <col span="1" style="width: 20%;">
  </colgroup>
<tbody><tr>
  <th>Name</th>
  <th>Role</th>
  <th>Summary</th>
  <th>Project Website</th>
  <th><a href="#grant" style="color:white;">Year</a></th>
</tr>

<tr>
  <td>NSF IIS-1453580: </td>
  <td>PI</td>
  <td>CAREER: A Data-Driven Network Inference Framework for Context-Conditioned Protein Interaction Graphs </td>
  <td> <a href='context_graph/index.htm'>Project Website</a> </td>
  <td>2015-2020</td>
</tr>


<tr>
  <td>NSF CNS-1619098: </td>
  <td>PI</td>
  <td>TWC:Small: Automatic Techniques for Evaluating and Hardening Machine
Learning Classifiers in the Presence of Adversaries </td>
  <td> <a href='16SecureLearning/index.htm'>Project Website</a> </td>
  <td>2016-2019</td>
</tr>
<tr>
  <td>ONR: </td>
  <td>CO-PI</td>
  <td>Deep Learning of Passage Structure for Scalable Semantic Discovery </td>
  <td> <a href=''>Project Website</a> </td>
  <td>2015-2018</td>
</tr>
<tr>
  <td>NSF CNS-1441875: </td>
  <td>co-PI</td>
  <td>PI Meeting for Secure and Trustworthy
Cyberspace: Matching Meeting Attendees Through Text Mining of their Scientific Reports. </td>
  <td> <a href='14Text_matchPI/index.htm'>Project Website</a>  </td>
  <td>2014-2015</td>
</tr>

</tbody>
</table>


        <table cellpadding="3" cellspacing="5" >
      <tr> <td>
<i><b>OLD legacy research sites</b></i>

  <li>
 [ <a href="http://deeplearning4discrete.net" target="_blank">Our deep learning tools on  discrete data types</a>]
  </li>
  <li>
 [ <a href="http://securemachinelearning.org" target="_blank">Our tools for making machine learning robust and trustworthy</a>]
  </li>
  <li>
 [ <a href="http://deeplearning4biomed.org" target="_blank">Our deep learning tools for analyzing sequential data types from bio-medicine</a>]
  </li>
 <li>
 [ <a href="http://jointggm.org" target="_blank">Our tools for learning dependency networks from heterogeneous samples (by joint structure learning of multiple Graphical Models)</a>]
  </li>
</td>
</tr>

</tbody>
</table>

     <div align="center"><span style="" lang="EN-US">
                       <hr align="left" size="1">      </span></div>


 <h4><font color="brown"><font size=+1>Representative publications:</font></font></h4>

<a name="topic"></a>

<table id="datatab2" summary="Representative publications: " border="1">
   <colgroup>
       <col span="1" style="width: 25%;">
       <col span="1" style="width: 75%;">
     </colgroup>
<tbody><tr>
  <th style="width: 22%;">Category</th>
  <th>Representative Papers</th>
</tr>
<tr>
  <td valign="top"> <br> <h5> Overall</h5>  </td>
  <td> <ul>
  	<li>Full <a href="papers.htm">Publications</a> list ; </li> <br>
    <li>[ <a href="http://scholar.google.com/citations?user=eXKdSu0AAAAJ&hl=en"
        target="_blank" >PI GoogleScholar Page</a> ] </li>    </ul>
       <img src="pics/Qsummary.png" height="325" border="0">  <br><br>
</td>
</tr><br><br>
<td valign="top"> <br> <h5>Trustworthy Machine Learning </h5> <br>
 <li>
 [ <a href="http://securemachinelearning.org" target="_blank">Our tools for making machine learning robust and trustworthy</a>]
  </li>
</td>
 <td>
           <img src="pics/trustworthyML.png" height="325" border="0">
       <img src="pics/trustMLTimeline.png" height="325" border="0">

 <ul>
 <li>"Feature Squeezing: Detecting Adversarial Examples in Deep Neural Networks",NDSS 2018 (<a href="https://arxiv.org/abs/1704.01155">PDF</a>)</li>
  <li>"Black-box Generation of Adversarial Text Sequences to Evade Deep Learning Classifiers", 2018 IEEE Security and Privacy Workshops (SPW), Ji Gao, Jack Lanchantin, Mary Lou Soffa, Yanjun Qi, (<a href="https://github.com/QData/deepWordBug/blob/master/about/Ji2017_EvadeNLP-extended.pdf">PDF</a>)</li>
<li>"Automatically Evading Classifiers",NDSS 2016    (<a href="paperA14/2016-evade_classifier.pdf">PDF</a>)(<a href="paperA14/2016-evade-ndsst.pdf">Talk</a>)
</li>
 <li>"A Theoretical Framework for Robustness of (Deep) Classifiers Against Adversarial Samples", ICLR-17 (<a href="https://arxiv.org/abs/1612.00334">Arxiv</a>)</li>
<li>"Feature Squeezing: Detecting Adversarial Examples in Deep Neural Networks", (<a href="https://arxiv.org/abs/1704.01155">Arxiv</a>)</li>
<li>"DeepCloak: Masking Deep Neural Network Models for Robustness against Adversarial Samples",ICLR-17 workshop (<a href="https://arxiv.org/abs/1702.06763">Arxiv</a>)</li>
<li>"Adversarial Playground: A Visualization Suite for Adversarial Sample Generation", Norton, Andrew and Qi, Yanjun,
  (<a href="http://arxiv.org/abs/1706.01763">Arxiv</a>)</li>
</ul>
</td>
</tr>
<tr>
  <td valign="top"> <br><h5>(Deep) Representation Learning on sequential data</h5> <br><font face="Arial,Helvetica" color="#990000" size=-1>(genome or epigenomic data, product review text, bio-reports text, protein sequence strings, etc.)</font> <br><br>
  <li>
 [ <a href="http://deeplearning4biomed.org" target="_blank">Our Tools for Mining Sequential Bio-Data</a>]
  </li>

<li>[ <a href="http://deeplearning4discrete.net" target="_blank">Our deep Learning tools on Discrete Structures</a>]</li>

  </td>
  <td>
         <img src="pics/deepChrome.png" height="325" border="0">
       <img src="pics/deepChromeTimeline.png" height="325" border="0">
  	<ul>
  	<li>"Attend and Predict: Understanding Gene Regulation by Selective Attention on Chromatin", at (NeurIPS 2017 )  (<a href="https://papers.nips.cc/paper/7255-attend-and-predict-understanding-gene-regulation-by-selective-attention-on-chromatin.pdf">PDF</a>)</li>
<li>
"DeepDiff: Deep-learning for predicting Differential gene expression from histone modifications", Bioinformatics 2018 (<a href="https://arxiv.org/pdf/1807.03878.pdf">PDF</a>)
</li>
<li>
"DeepChrome: Deep-learning for predicting gene expression from histone modifications", Bioinformatics 2016 (<a href="http://arxiv.org/pdf/1607.02078.pdf">PDF</a>)
</li>
<li>"Memory Matching Networks for Genomic Sequence Classification", ICLR2017, (<a href="https://arxiv.org/abs/1702.06760">Arxiv</a>)
(<a href="paperA14/2017-ICLR-Poster-MeMo.pdf">Poster</a>)</li>
<li>"GaKCo: a Fast GApped k-mer string Kernel using COunting", ECML17, (<a href="http://arxiv.org/abs/1704.07468">Arxiv</a>)
  </li>
<li>
"Deep Motif Dashboard: Visualizing and Understanding Genomic Sequences Using Deep Neural Networks", PSB17  (<a href="http://arxiv.org/abs/1608.03644">Arxiv</a>)
</li>
<li>
"Deep Motif: Visualizing Genomic Sequence Classifications", ICLR2016, the International Conference on Learning Representations   (<a href="http://arxiv.org/pdf/1605.01133.pdf">Arxiv</a>)
</li>
<li>"MUST-CNN: A Multilayer Shift-and-Stitch Deep Convolutional Architecture for Sequence-based Protein Structure Prediction", <i>AAAI 2016.</i> (<a href="paperA14/2016-MustCnn.pdf">PDF</a>)    (<a href="paperA14/2016-AAAIt.pdf">Talk</a>)
 </li>
<br>
   	<li>"Unsupervised Feature Learning by Deep Sparse Coding", SDM 2014. (<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/2014_sdm_DeepSC.pdf">PDF</a>)(<a href="http://www.cs.virginia.edu/yanjun/paperA14/2014-sdm-deepsc-talk.pdf">Talk</a>) </li>

   	<li>"Deep Learning for Character-based Information Extraction",ECIR 2014, (<a href="http://www.cs.cmu.edu/%7Eqyj/zhSenna/2014_ecir2014_full.pdf">PDF</a>)
(<a href="http://www.cs.cmu.edu/%7Eqyj/zhSenna/moreSenna_supplement.pdf">SupplementaryDoc</a>)(<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/2014_ecir_deep.pdf">Talk</a>)</li>

 		<li>"Learning the Dependency Structure of Latent Factors",NIPS 2012, (<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/12-slfa.pdf">PDF</a>)(<a href="http://www.cs.virginia.edu/yanjun/paperA14/2012_SLFA_NIPS.pdf">Talk</a>)</li>

	<li>"A unified multitask architecture for predicting local protein properties", <i> PLoS ONE 2012</i>(<a href="./paperA14/2012-DeepProtein.pdf">(PDF)</a>)</li>

	<li>"Sentiment Classification with Supervised Sequence Encoder", ECML 2012, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/sse_ecml12_camready.pdf"
  rel="nofollow" style="color: blue;text-decoration:underline;">PDF</a>)(<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/12-talk-ecml2012-qyj.pdf">Talk</a>) </li>

  <li>"Sentiment Classification Based on
Supervised Latent n-gram Analysis",CIKM 2011, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/11-cikm.pdf"
  rel="nofollow" style="color: blue;text-decoration:underline;">PDF</a>)(<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/11-talk-cikm11.pdf">Talk</a>)</li>

<li>"Semi-Supervised
  Abstraction-Augmented String Kernel for Multi-Level Bio-Relation
  Extraction", ECML 2010, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/ecml10.pdf"
  rel="nofollow" style="color: blue; text-decoration:
  underline;">PDF</a>)(<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/ecml10-Talk-online.pdf"
  rel="nofollow" style="color: purple;
  text-decoration:underline;">Talk</a>) </li>

<li>Semi-Supervised Bio-Named Entity Recognition with Word-Codebook
  Learning", SDM 2010, (<a  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/bNER-sdm10.pdf"
  rel="nofollow"><font color="#0000ff">PDF</font></a>)
</li>

<li>"Polynomial Semantic Indexing", NIPS 2009, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/2009_psi_nips.pdf"  rel="nofollow">PDF</font></a>)</li>

<li>"Semi-Supervised Sequence Labeling with Self-Learned Feature", ICDM 2009, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/selfFea-icdm09-c5.pdf"
  rel="nofollow"><font color="#0000ff">PDF</font></a>)(<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/ICDM09-slidesv2.pdf"
  rel="nofollow"><font color="#0000ff">Talk</font></a>)</li>

<li>"Combining Labeled and Unlabeled Data for Word-Class Distribution" , CIKM 2009, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/wcdl-cikm09.pdf"
  rel="nofollow"><font color="#0000ff">PDF</font></a>)</li>
  </ul>
  </td>
</tr>
<tr>
  <td valign="top"> <br><h5>Learning Graph from Data</h5> <br>
    <font face="Arial,Helvetica" color="#990000" size=-1>(Helping researchers effectively translate aggregated data into knowledge that take the form of graphs)</font> <br><br>
     <li>
 [ <a href="http://jointggm.org" target="_blank">Our tools for learning networks from heterogeneous samples (by joint structure learning of Gaussian Graphical Models)</a>]
  </li>

  </td>
  <td>
       <img src="pics/jointNets.png" height="325" border="0">
       <img src="pics/jointNetTimeline.png" height="325" border="0">
    <ul>
     <li>"A fast and scalable joint estimator for integrating additional knowledge in learning multiple related sparse gaussian graphical models.", ICML (2018)(<a href="http://proceedings.mlr.press/v80/wang18f.html">PDF</a>)</li>
     <li>"Fast and Scalable Learning of Sparse Changes in High-Dimensional Gaussian Graphical Model Structure", AISTATS (2018)(<a href="https://arxiv.org/abs/1710.11223">PDF</a>)</li>
      <li>"A Fast and Scalable Joint Estimator for Learning Multiple Related Sparse Gaussian Graphical Models", AISTATS (2017)(<a href="http://arxiv.org/abs/1702.02715">Arxiv</a>)(<a href="paperA14/2017_JEM_combined.pdf">PDF</a>)</li>
      <li> "A constrained L1 minimization approach for estimating multiple Sparse Gaussian or Nonparanormal Graphical Models", (2017) Machine Learning Journal ; (<a href="https://arxiv.org/abs/1605.03468">PDF
      </a>)</i>
        <li>"Learning the Dependency Structure of Latent Factors",NIPS 2012, (<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/12-slfa.pdf">PDF</a>)(<a href="http://www.cs.virginia.edu/yanjun/paperA14/2012_SLFA_NIPS.pdf">Talk</a>)</li>
        <li>"Transfer String Kernel for Cross-Context DNA-Protein Binding Prediction",@ (TCBB) (<a href="http://arxiv.org/abs/1609.03490">Arxiv</a>) </li>
      <li>"Protein Complex Identification by Supervised Graph Clustering", Bioinformatics 2008, (<a
  href="http://www.cs.cmu.edu/%7Eqyj/paper_CMU/ISMB08_qyj.pdf"
  rel="nofollow">PDF</a>)(<a
  href="http://www.cs.cmu.edu/%7Eqyj/SuperComplex/sharefiles/complex-Talk-online.pdf"
  rel="nofollow">Talk</a>)</li>
    </ul>
  </td>
</tr>
<tr>

<tr>
  <td><h5>Add Weak Supervision in Machine Learning</h5></td>
  <td>
  	<ul>
      <img src="pics/weaksupervised.png" height="325" border="0"> 
      <li> "Curriculum Labeling: Self-paced Pseudo-Labeling for Semi-Supervised Learning", P Cascante-Bonilla, F Tan, Y Qi, V Ordonez, arXiv:2001.06001 </li>

<li>" Yanjun Qi, B. Bai, X. Ning, and P. Kuksa. Systems and methods for Semi-supervised Relationship Extraction,  Apr. 2014. </li>

  		<li>"Semi-Supervised Convolution
  Graph Kernels for Relation Extraction", SDM 2011,
(<a
  href="http://www.cs.cmu.edu/%7Eqyj/papersA08/SSCG-sdm11.pdf"
  rel="nofollow" style="color: blue;text-decoration:
  underline;">PDF</a>) (<a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/11-talk-sdm11-sgk.pdf">Talk</a>)
	</li>
  		<li>"Semi-Supervised Multi-Task Learning for Predicting Interactions between HIV-1 and Human Proteins", ECCB 2010, (<a href="http://www.cs.cmu.edu/%7Eqyj/HIVsemi/hivPPI-eccb10.pdf" rel="nofollow" style="color: blue; text-decoration: underline;">PDF</a>)
  			(<a
    href="http://www.cs.cmu.edu/%7Eqyj/HIVsemi/eccb10-Talk-online.pdf"
    rel="nofollow" style="color:
    purple;text-decoration:underline;">Talk</a>)
  		</li>

  	</ul>
  </td>
</tr>
<tr>
  <td><h5>Machine Learning for Health</h5></td>
  <td>
  	<ul>
                  <img src="pics/interpretDeep.png" height="325" border="0"> 

  		<li>"Kernelized Information-Theoretic Metric Learning for Cancer Diagnosis  using
  High-Dimensional Molecular Profiling Data", TKDD 2016, (<a href="paperA14/2015-TKDD.pdf">PDF</a>)  </li>
  		<li>"MAPer: A Multi-scale Adaptive Personalized Model for Temporal Human Behavior Prediction ",CIKM 2015,
     (<a href="paperA14/2015-maper.pdf" rel="nofollow">PDF</font></a>)
     (<a href="paperA14/2015-MAPerCIKM.pdf">Talk</a>) </li>
  		<li>"Causal Analysis of Inertial Body Sensors for Enhancing Gait Assessment Separability towards Multiple Sclerosis Diagnosis", IEEE Body Sensor Network (BSN) 2015, (<a href="paperA14/2015-casualSensor.pdf">PDF</a>) (<a href="paperA14/2015-casualT.pdf">Talk</a>)   </li>
   		<li>"Piecewise Linear Dynamical Model for Action Clustering from Real-World Deployments of Inertial Body Sensors", BodyNets 2014,  (<a
  href="paperA14/201408_bodynets.pdf">PDF</a>) (<a
  href="paperA14/2014-BodyNetsBestPaper.pdf">Talk</a>) </li>

  		<li>Retrieving Medical Records with sennamed: NEC Labs America at TREC 2012 Medical Record Track, 2012 Text Retrieval Conference, <a href="http://www.cs.cmu.edu/%7Eqyj/papersA08/12_sennamed.pdf">(PDF)</a></li>

  	</ul>
  </td>
</tr>
</tbody>
</table>


<br>




<div style="position: fixed; bottom: 10px; right:10px; width: 88px; height: 150px; background-color: #FFCF79;">
<a style="position: fixed; bottom:130px; right:10px;" href="#info">Info</a>
<a style="position: fixed; bottom:110px; right:10px;" href="#topic">Highlights </a>
<a style="position: fixed; bottom:90px; right:10px;" href="#grantNow">GrantNow</a>
<a style="position: fixed; bottom:70px; right:10px;" href="#grantThen">GrantThen</a>
<a style="position: fixed; bottom:30px; right:10px;" href="#top">Back2Top</a>
</div>



			<!-- #EndEditable --></div>
		<!-- End Content --></div>
	<!-- End Page Content -->
	<!-- Begin Footer -->
		<div id="footer">
		<p>
		<span class="style6">
			<div align="center"><span style="" lang="EN-US">
			</span></div>
			<p align="left"><i>Success
			is not final, failure is not fatal: it is the courage to continue that counts.</i>
			--- Winston Churchill</p></div>
		</span>
		</p>
	</div>
	<!-- End Footer -->
</div>
<!-- End Container -->

<!-- #EndTemplate -->

</body></html>
